### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by alejandro5042**
## Why do we need this?

We want this ReviewBoard userscript on GitHub.

## What does this change do?

This is the initial commit of the userscript, containing all the source.

## How was this tested?

It's been in active use by 4-5 developers over the past week.

## Checklist

If applicable:

- [x] Runs in latest stable Chrome and Firefox without Javascript errors
- [x] Does not report any issues with `eslint` (run `npm run build` -- the PR build will also check this)
- [x] Increment version following [semver](https://semver.org/)
- [x] All `@require` and `@resource` URLs must have SRI ([use this hashing tool](https://www.srihash.org/))
- [x] Blocks of code are commented
- [ ] Included a screenshot in the PR
- [x] Added docs to `README.md`
- [ ] Linked issues

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Alejandro Barreto
```
Comment (zhindes) [inline LICENSE.txt:3]: Make this National Instruments, not you.

File: src/ni-reviewboard.user.js [hunk 2]
```diff
@@ -0,0 +1,610 @@
...
+// @updateURL    https://rebrand.ly/update-ni-reviewboard-user-js
+// @include      https://review-board.natinst.com/*
+// @run-at       document-start
+// @grant        GM_addStyle
+// @grant        GM_getValue
+// @grant        GM_setValue
+// @grant        GM_addValueChangeListener
+// @grant        GM_registerMenuCommand
+// @require      https://cdn.jsdelivr.net/npm/sweetalert2@9.13.1/dist/sweetalert2.all.min.js#sha384-8oDwN6wixJL8kVeuALUvK2VlyyQlpEEN5lg6bG26x2lvYQ1HWAV0k8e2OwiWIX8X
+// @require      https://gist.githubusercontent.com/alejandro5042/af2ee5b0ad92b271cd2c71615a05da2c/raw/easy-userscripts.js?v=70#sha384-jDwgCHbDuz3Crr4lWqnjJUbjVoZeouMp9v6zAHkV8DAPpiaXoO+O2LQJyJ9AHYeV
```
[the remark this answers is not in the document]
Comment (alejandro5042) [reply src/ni-reviewboard.user.js:16]: Added MIT license to the library and officially copyrighting it for NI.

File: LICENSE.txt (hunk 1 above)
Comment (alejandro5042) [reply LICENSE.txt]: Done!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+Primarily for use at NI. If you also use Review Board, it is possible that this script can also help you too (with modifications).
+
+## Getting Started
+
+1. Install a userscripts extension — such as [Tampermonkey](https://tampermonkey.net/) or [Violentmonkey](https://violentmonkey.github.io/) — and refresh this page
+2. Install this [userscript](https://github.com/ni/reviewboard-userscripts/raw/master/src/ni-reviewboard.user.js)
+3. Refresh Review Board
+
+## What does it do?
+
```
Comment (humberto-garza) [inline README.md:12]: Missing?

**Pull request merged**